### DISCUSSION TITLE
Yolov5 and Yolov8 .NET 6 interface on Nuget

### DISCUSSION THREAD

**Discussion by sstainba** [ultralytics/ultralytics#1015] [Show and tell]
For any other .NET devs out there, I created a package to run predictions on Yolov5/v8 models (in ONNX format) for .NET 6.  It's on Nuget at https://www.nuget.org/packages/Yolov8.Net

**Comment by AyushExel** [comment]
Wow! Nice work. I'd love to feature this here https://github.com/ultralytics/ultralytics/tree/main/examples . What do you think?

**Reply by sstainba** [reply]
Fine by me.  :-)

**Reply by AyushExel** [reply]
Would you like to make PR adding this to the list of examples so that we can attribute it to you?
